src/core/src/sys/cells.rs

```diff
@@ -276,6 +276,30 @@ impl UnitCell {
         }
     }
 
+    /// Return a new vector that lies inside the simulation cell.
+    ///
+    /// If `vect` is already in the cell or the `CellShape` is
+    /// `Infinite`, the resulting vector is identical to `vect`.
+    pub fn get_wrapped_vector(&self, vect: &Vector3D) -> Vector3D {
```

Comment:
Why is this needed? Couldn't the `wrap_vector` function be used instead? i.e. replace

```rust
let wrapped = cell.get_wrapped_vector(original);
```

by 

```rust
let mut wrapped = original.clone();
cell.wrap_vector(&mut wrapped);
```